Jenkinsfile

```diff
@@ -0,0 +1,22 @@
+pipeline {
+    agent {label 'jenkins-slave-archita'}
+
```

Comment:
this branch has to be based on the ruby  branch and just have an extra stage called merge